tests/daemon/test-daemon.c

```diff
@@ -107,7 +107,7 @@ static void
 start_mock_logind_service (Fixture *fixture)
 {
   fixture->logind_mock = g_subprocess_new (G_SUBPROCESS_FLAGS_STDOUT_PIPE, NULL,
-                                           "python", "-m", "dbusmock",
+                                           "python3", "-m", "dbusmock",
```

Comment:
Not sure what's the best way to pick up the detected Python executable here. Perhaps compile with `-DPYTHON_PATH="\"$(PYTHON)\""` and use `PYTHON_PATH` here?